### DISCUSSION TITLE
Restore DB 1.15 to new instalation 2.0

### DISCUSSION THREAD

**Discussion by ilicilho** [kimai/kimai#4011] [On-Premise: Installation & Updates]
Hi there.

I have a backup from my install version 1.15. Today my server was corrupted. It´s possible to install a new version and restore this DB backup?

**Comment by kevinpapst** [comment]
Yes. As described in the documentation, thanks for reading it upfront https://www.kimai.org/documentation/backups.html

**Reply by ilicilho** [reply]
Sorry, Kevin. I Read the docs but not find any mention to restore a old version DB(1.15.5 stable (Ayumi)) in a new version.
BTW, I install the latest version of Kimai and restore DB, I see error 500.
